base_report_to_printer/wizards/printing_printer_update_wizard.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2009 Albert Cervera i Areny <albert@nan-tic.com>
+# Copyright (C) 2011 Agile Business Group sagl (<http://www.agilebg.com>)
+# Copyright (C) 2011 Domsense srl (<http://www.domsense.com>)
+# Copyright (C) 2014 Camptocamp (<http://www.camptocamp.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+
+class PrintingPrinterUpdateWizard(models.TransientModel):
+    _name = "printing.printer.update.wizard"
```

Comment:
don't forget add ACL and rules for this model

Reply:
Is in security.xml. There was something wrong with my commit when you checked it, missing some files. I fixed it already.